gm4_area_markers/data/area_markers/functions/main.mcfunction

```diff
@@ -0,0 +1,11 @@
+#run from pulse_check
+
+#create marker
+execute as @e[type=item_frame,tag=!gm4_area_marker] if data entity @s {Item:{tag:{gm4_area_marker:1b}}} unless data entity @s {Item:{tag:{display:{Name:"{\"translate\":\"%1$s%3427655$s\",\"with\":[\"Area Marker Tier 1\",{\"translate\":\"item.gm4.area_markers.tier_1\"}],\"italic\":\"false\"}"}}}} unless data entity @s {Item:{tag:{display:{Name:"{\"translate\":\"%1$s%3427655$s\",\"with\":[\"Area Marker Tier 2\",{\"translate\":\"item.gm4.area_markers.tier_2\"}],\"italic\":\"false\"}"}}}} unless data entity @s {Item:{tag:{display:{Name:"{\"translate\":\"%1$s%3427655$s\",\"with\":[\"Area Marker Tier 3\",{\"translate\":\"item.gm4.area_markers.tier_3\"}],\"italic\":\"false\"}"}}}} at @s positioned ^ ^ ^0.3 positioned ~ ~-.3 ~ unless entity @e[type=area_effect_cloud,tag=gm4_area_marker,distance=..0.1] run function area_markers:marker/create
```

Comment:
All of these `unless data` checks seem unnecessary. If the item frame doesn't have the tag, that means it wouldn't have the item because it would be killed if it did have the item and also the tag. Note: my code suggestion also has an optimization that only item frames near players are checked
```suggestion
execute at @a as @e[type=item_frame,distance=..16,tag=!gm4_area_marker,nbt={Item:{tag:{gm4_area_marker:1b}}}] at @s positioned ^ ^ ^0.3 positioned ~ ~-.3 ~ unless entity @e[type=area_effect_cloud,tag=gm4_area_marker,distance=..0.1] run function area_markers:marker/create
```